src/ui/foundation/MainLayout/data.tsx

```diff
@@ -8,73 +8,65 @@ import {
 } from '@mui/icons-material';
 import { TreeItem, TreeView } from '@mui/lab';
 import { IconButton, TextField, Typography } from '@mui/material';
-import { NavLink, useNavigate } from 'react-router-dom';
+import { NavLink, useNavigate, NavigateFunction } from 'react-router-dom';
+import React from 'react';
 
 import { pathFor, PathName } from '@/ui/utilities/paths';
+import { useVaultPages } from '@/ui/hooks/query/useVaultPages';
+import { parsePathListToTree, TreeNode } from '@/lib/path/pathListToTree';
+
+function createTreeItem(
```

Comment:
This is a react component, so it should follow react's naming convention for a component: `function TreeItem()`